reference-implementation/__tests__/json/packages-via-trailing-slashes.json

```diff
@@ -6,7 +6,8 @@
       "lodash-dot": "./node_modules/lodash-es/lodash.js",
       "lodash-dot/": "./node_modules/lodash-es/",
       "lodash-dotdot": "../node_modules/lodash-es/lodash.js",
-      "lodash-dotdot/": "../node_modules/lodash-es/"
+      "lodash-dotdot/": "../node_modules/lodash-es/",
+      "mapped/path/": "https://github.com/WICG/import-maps/issues/207/"
```

Comment:
Could you add an entry of `"mapped/"` here, and `"mapped/backtrack"` to the `expectedResults` below?
This is to test that the backtracking failure at a more specific key doesn't allow fallback to less specific key -- i.e. `"mapped/path/../backtrack"` can't be resolved while `"mapped/backtrack"` can be resolved.